helm/grafana/values.yaml

```diff
@@ -44,6 +44,10 @@ image:
   repository: grafana/grafana
   tag: 4.4.1
 
+grafana_watcher:
```

Comment:
Use camel case instead of underscore as suggested by Helm chart best practices and to be consistent.